docs/source/tutorial/package_a_project.md

```diff
@@ -4,25 +4,96 @@ This section explains how to build your project documentation, and how to bundle
 
 Kedro also has an advanced feature which supports packaging on a pipeline level allowing you share and reuse pipelines across projects! To read more about this please look at the [section on micro-packaging](../nodes_and_pipelines/micro_packaging.md).
 
-
 ## Add documentation to your project
 
-Kedro uses the [Sphinx framework](https://www.sphinx-doc.org) and creates a `docs` directory that builds a basic template for project-specific documentation. We recommend that you add your project-specific documentation as markdown in `docs/source`
+Kedro uses the [Sphinx framework](https://www.sphinx-doc.org) to build any documentation that you create about your project.
+
+To install a compatible version of Sphinx, run the following from the project root directory:
+
+```bash
+pip install sphinx
+```
+
+### Set up the Sphinx project files
+Navigate to the `docs` directory of your Kedro project. Before Kedro version 0.19, the `docs/source` subdirectory was pre-populated with two Sphinx configuration files (`conf.py`, and `index.rst`) which the `kedro build-docs` command needed.
 
-If you want to customise your documentation beyond the basic template, refer to the [Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/configuration.html) for details of how to extend `docs/source/conf.py`.
+From Kedro version 0.19 that command is deprecated, and those files will no longer exist. If you have used an earlier version of Kedro to create your project, and want to follow the following instructions to build the documentation without using `kedro build-docs`, you should delete the two files and the `source` subdirectory so that your project's `docs` directory is empty.
 
-Once you have added any documentation you need, run the following from the project root directory:
+Navigate to the empty `docs` directory in the terminal, and run the following command:
 
 ```bash
-kedro build-docs --open
+sphinx-quickstart
```

Comment:
This only works if you have an `index.rst` though, right?

Reply:
I think my main issue with using `kedro new` to create the sphinx files is that they are way more complex than what `sphinx-quickstart` creates, and we are tied to Sphinx 3.4.3 rather than Sphinx >6.0 (with some complications over Jinja). I don't know how much work it would be to migrate Kedro up to a more recent version of Sphinx but I don't want to go there :) 

I'd rather we just unhook Kedro and push the user towards a basic Sphinx usage if they want it. Like you say, they don't have to use Sphinx, and personally I don't think they are best advised to unless there's good reason.

Reply:
I see how it's good to start from scratch with `conf.py`. The `make html` is not mandatory but it's what most tutorials use anyway, so it's good to stay in line with that. Let's keep the `sphinx-quickstart` workflow.